tests/jwt_auth/test_jwt_middleware.py

```diff
@@ -199,3 +205,13 @@ def test_token_without_user_id(self):
             response.json(),
             {"user_id": None, "jwt_error": JWTError.user_not_found.value},
         )
+
+    def test_visible_paths(self):
```

Comment:
We should update this to `test_excluded_paths`.

Reply:
I forgot to change that. Thanks.